File: Java/Product.java
class Product 
{
    //atribut private
    private String id;         
    private String name;    
    private String brand;    
    private String price;

    //constructor
    public Product()
    {
        this.id = "";
        this.name = "";
        this.brand = "";
        this.price = "";
    }
    public Product(String id, String name, String brand, String price)
    {
        this.id = id;
        this.name = name;
        this.brand = brand;
        this.price = price;
    }

    //setter
    public void setId(String id)
    {
        this.id = id;
    }
    public void setName(String name)
    {
        this.name = name;
    }
    public void setBrand(String brand)
    {
        this.brand = brand;
    }
    public void setPrice(String price)
    {
        this.price = price;
    }

    //getter
    public String getId()
    {
        return this.id;
    }
    public String getName()
    {
        return this.name;
    }
    public String getBrand()
    {
        return this.brand;
    }
    public String getPrice()
    {
        return this.price;
    }
}

File: C++/Product.cpp
/*
    Saya Sabila Rosad NIM 2106000 mengerjakan latihan 2
    dalam mata kuliah Desain Pemrograman Berorientasi Objek
    untuk keberkahanNya maka saya tidak melakukan
    kecurangan seperti yang telah dispesifikasikan
*/
// library
#include <iostream>
#include <string>

using namespace std;

// deklarasi class
class Product
{
    // atribut private
private:
    string id;
    string name;
    string brand;
    string price;

public:
    // constructor
    Product()
    {
        this->id = "";
        this->name = "";
        this->brand = "";
        this->price = "";
    }
    Product(string id, string name, string brand, string price)
    {
        this->id = id;
        this->name = name;
        this->brand = brand;
        this->price = price;
    }

    // setter
    void setId(string id)
    {
        this->id = id;
    }
    void setName(string name)
    {
        this->name = name;
    }
    void setBrand(string brand)
    {
        this->brand = brand;
    }
    void setPrice(string price)
    {
        this->price = price;
    }

    // getter
    string getId()
    {
        return this->id;
    }
    string getName()
    {
        return this->name;
    }
    string getBrand()
    {
        return this->brand;
    }
    string getPrice()
    {
        return this->price;
    }

    // destructor
    ~Product()
    {
    }
};